src/admin-views/aggregator/origins/gcal.php

```diff
@@ -79,14 +79,14 @@ class="tribe-bumpdown-trigger tribe-bumpdown-permanent tribe-bumpdown-nohover tr
 <?php
 $field              = (object) array();
 $field->label       = __( 'URL:', 'the-events-calendar' );
-$field->placeholder = __( 'https://calendar.google.com/calendar/ical/example/basic.ics', 'the-events-calendar' );
-$field->help        = __( 'Enter the url for the Google Calendar feed you wish to import.', 'the-events-calendar' );
+$field->placeholder = 'https://calendar.google.com/calendar/ical/example/basic.ics';
```

Comment:
any reason not to let this example text be translatable? I don't think Google uses alternate domains internationally, but, would need someone outside the US to verify

Reply:
actually - this should be translatable - the words "example" and "basic" in the example URL should ideally be translated